tests/ui/parser/deli-ident-issue-1.rs

```diff
@@ -0,0 +1,27 @@
+// ignore-tidy-trailing-newlines
```

Comment:
Why is missing trailing newline important for this test?

Reply:
Removed it.